src/applications/avs/components/ErrorBoundary.jsx

```diff
@@ -34,7 +34,7 @@ class ErrorBoundary extends React.Component {
       </div>
     );
 
-    return hasError ? <ErrorMessage /> : <>{children}</>;
+    return hasError || !children ? <ErrorMessage /> : <>{children}</>;
```

Comment:
This change allows us to render the error page when there is an error in the data loader.